lib/healthlocker.ex

```diff
@@ -14,6 +14,7 @@ defmodule Healthlocker do
       supervisor(Healthlocker.Endpoint, []),
       # Start your own worker by calling: Healthlocker.Worker.start_link(arg1, arg2, arg3)
       # worker(Healthlocker.Worker, [arg1, arg2, arg3]),
+      worker(Segment, [Application.get_env(:segment, :write_key)])
```

Comment:
[`System.get_env`](https://hexdocs.pm/elixir/System.html#get_env/1) fetches an actual environment variable, which is managed by the OS process. 

[`Application.get_env`](https://hexdocs.pm/elixir/Application.html#get_env/2) is an Elixir helper which is basically just doing an internal key/value lookup. You can see this key is set in [`config.exs`](https://github.com/healthlocker/healthlocker/blob/master/config/config.exs#L37). 

The pattern seems to be, load all your environment variables in the config, but then access all your configuration through `Application`.